scripts/k6/user-scenario-test.js

```
@@ -0,0 +1,51 @@
+    let homeRes = http.get('https://api.movie-bookie.shop/api/events/home', {
+        headers: {
+            'accept': '*/*',
+            'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJtMDIwMjAyQG5hdmVyLmNvbSIsImNhdGVnb3J5IjoicmVmcmVzaCIsImlhdCI6MTc1MDU4OTM3NCwiZXhwIjoxNzUxNzk4OTc0fQ.dgZkEKvI2irgmGr9c5GwaQF18brAAUMyEYfQUg3YoBXB-sX5JYqgcJ8341UANgmoajtoUUE-Hj9SRcmDIReOQQ'
+        }
+    });
+    check(homeRes, {
+        '홈화면 조회 성공': (r) => r.status === 200
+    });
+
+    // 2. 이벤트 신청
+    let registerRes = http.post('https://api.movie-bookie.shop/api/events/191/register', null, {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'registerRes' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)